### ISSUE TITLE
Need to add active flag for users

### ISSUE THREAD

**Issue by shelleydoljack** [sul-dlss-deprecated/folio-migration-tasks#99]
```
"active": {
            "description": "A flag to determine if the user's account is effective and not expired. The tenant configuration can require the user to be active for login. Active is different from the loan patron block",
            "type": "boolean"
          },
```
should be added for each user.

**Comment by shelleydoljack** [comment]
Maybe this already is set and somehow we have old users loaded without the active flag? I just checked the json for a user that was created today from our folio user load (d39ddfa7-3885-44f7-a46b-3f069726514d ) and it does have `active: true`.

**Comment by shelleydoljack** [comment]
I think it is in TsvUserTaskHelpers where we are not setting the active flag: https://github.com/sul-dlss/folio-tasks/blob/c4c9ed1e81b1df6e91b3d3d56fab681dd08d42b7/tasks/helpers/tsv_user.rb#L113-L124

**Comment by jgreben** [comment]
Fixed by #101
